.github/ISSUE_TEMPLATE/long_report.yml

```diff
@@ -26,7 +26,6 @@ body:
       options:
         - always
         - here
-        - stale
```

Comment:
The previous commit b8e6fb18222316c551157fa1089ac35eeb47d1a8 updated the template file so the static dropdown was updated